test/fixture/samlp-server.js

```diff
@@ -220,6 +220,20 @@ passport.use('samlp-with-utf8', new Strategy(
   })
 );
 
+passport.use('samlp-with-ISO', new Strategy(
+  {
+    path: '/callback',
+    cert: pemToCert(fs.readFileSync(path.join(__dirname, '../test-auth0.pem'))),
+    checkExpiration: false, // we are using a precomputed assertion generated from a sample idp feide
+    checkAudience: false,
+    checkDestination: false,
+    checkRecipient: false
```

Comment:
don't you want to pass the default_encoding?

Reply:
Added a test without default encoding on one with default encoding.